requirements.txt

```diff
@@ -355,7 +359,6 @@ djangorestframework==3.15.2 \
     --hash=sha256:36fe88cd2d6c6bec23dca9804bab2ba5517a8bb9d8f47ebc68981b56840107ad
     # via
     #   -r requirements.in
-    #   django-helsinki-notification
```

Comment:
This removed line should be still present...? 

```bash
$ pip-compile --generate-hashes --strip-extras requirements.in
$ git diff
``` 


```diff
diff --git a/requirements.txt b/requirements.txt
index d34b99d..b3b9baa 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -359,6 +359,7 @@ djangorestframework==3.15.2 \
     --hash=sha256:36fe88cd2d6c6bec23dca9804bab2ba5517a8bb9d8f47ebc68981b56840107ad
     # via
     #   -r requirements.in
+    #   django-helsinki-notification
     #   django-parler-rest
     #   drf-nested-routers
     #   drf-oidc-auth
```

Reply:
Running the same command, I do receive warning and it does not add that given line:
`WARNING: django-helsinki-notification 0.0.0 does not provide the extra 'rest_framework'`

Reply:
Ended up being problem with pip 24.0. Updated to 25.1 fixed the issue and restored that missing line. Too new pip on the other hand broke pip-tools compatibility.